DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -461,6 +474,95 @@ internal DiscordGuild()
 
         #region Guild Methods
 
+        /// <summary>
+        /// Gets a guild sticker by it's Id.
+        /// </summary>
+        /// <param name="stickerId">The Id of the sticker to get.</param>
+        /// <returns>The specificed sticker.</returns>
+        public Task<DiscordMessageSticker> GetStickerAsync(ulong stickerId)
+            => this.Discord.ApiClient.GetGuildStickerAsync(stickerId, this.Id);
+
+        /// <summary>
+        /// Creates a new sticker on the guild. Requires ManageStickers permission.
+        /// </summary>
+        /// <param name="name">The name of the sticker.</param>
+        /// <param name="description">The optional description of the sticker.</param>
+        /// <param name="emoji">The emoji to associate the sticker with.</param>
+        /// <param name="format">The file format the sticker is written in.</param>
+        /// <param name="file">The sticker.</param>
+        /// <returns>The created sticker.</returns>
+        public async Task<DiscordMessageSticker> CreateStickerAsync(string name, string? description, DiscordEmoji emoji, StickerFormat format, Stream file)
+        {
+            if (emoji.Id is not 0)
+                throw new InvalidOperationException("Only unicode emoji can be used for stickers.");
+
+            if (name.Length is > 30 or < 2)
+                throw new ArgumentOutOfRangeException(nameof(name), "Sticker name must be 2-30 characters.");
```

Comment:
These are inconsistent with our conditional format used throughout the rest of the library.